frontend/src/scenes/data-management/definition/DefinitionEdit.tsx

```diff
@@ -102,6 +103,35 @@ export function DefinitionEdit(props: DefinitionEditLogicProps): JSX.Element {
                     </Col>
                 </Row>
             )}
+            {hasTaxonomyFeatures && !isEvent && (
+                <Row gutter={[16, 24]} className="mt ph-ignore-input" style={{ maxWidth: 640 }}>
```

Comment:
Should really be using `LemonRow` instead of antd `Row` but probably off-topic to fix that in this PR

Reply:
Yep, not for this PR, I just followed the Rows above 😋